scripts/babel-relay-plugin/src/getBabelRelayPlugin.js

```diff
@@ -191,6 +192,11 @@ function getBabelRelayPlugin(
               console.log(error.message);
               console.log(error.stack);
             }
+            if (options && options.abortOnError) {
+              throw new Error(
+                'Aborting due to GraphQL validation/transform error(s).'
```

Comment:
Wouldn't it be useful to preserve the error message here? (Or just rethrow the original `error`?)